atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -145,27 +145,46 @@ public String toString() {
         }
     }
 
-    private static class RequestMetrics {
-        private final Counter totalRequests = new Counter();
-        private final Counter totalRequestExceptions = new Counter();
-        private final Counter totalRequestConnectionExceptions = new Counter();
+    private class RequestMetrics {
+        private final Meter totalRequests;
+        private final Meter totalRequestExceptions;
+        private final Meter totalRequestConnectionExceptions;
 
-        Counter getTotalRequests() {
-            return totalRequests;
+        RequestMetrics() {
```

Comment:
nit: this code can be de-duped by using the below method with an empty string (the metrics registry will happily handle them) - From MetricsRegistry.java:
```
    /**
     * Concatenates a class name and elements to form a dotted name, eliding any null values or
     * empty strings.
     *
     * @param klass    the first element of the name
     * @param names    the remaining elements of the name
     * @return {@code klass} and {@code names} concatenated by periods
     */
    public static String name(Class<?> klass, String... names) {
```